cmd/vulcan-aws-alerts/rds.go

```diff
@@ -77,28 +61,41 @@ func caCertificateRotation(target string, vulcanAssumeRoleEndpoint string, roleN
 
 					for _, instance := range result.DBInstances {
 						if instance != nil {
-							m := make(map[string]string)
-							m["AutoAppliedAfterDate"] = fmt.Sprintf("%s", aws.TimeValue(details.AutoAppliedAfterDate))
-							m["CurrentApplyDate"] = fmt.Sprintf("%s", aws.TimeValue(details.CurrentApplyDate))
-							m["Identifier"] = aws.StringValue(instance.DBInstanceIdentifier)
-							m["Account"] = target
-							m["Region"] = region
-							m["DBName"] = aws.StringValue(instance.DBName)
-							m["Engine"] = aws.StringValue(instance.Engine)
-							m["ARN"] = aws.StringValue(instance.DBInstanceArn)
-							rg.Rows = append(rg.Rows, m)
+							state.AddVulnerabilities(
+								report.Vulnerability{
+									AffectedResource: aws.StringValue(instance.DBInstanceArn),
+									Labels:           []string{"issue"},
+									Fingerprint:      helpers.ComputeFingerprint(details.AutoAppliedAfterDate),
+									Summary:          `Managed AWS databases using CA about to expire`,
+									Score:            report.SeverityThresholdHigh,
+									Description: `Due to the expiration of the AWS RDS CA, and to prevent downtime ` +
+										`in your applications, you should add the new CA to your clients using a ` +
+										`managed (i.e. RDS or Aurora) database through SSL/TLS and perform maintenance ` +
+										`on the affected database instances before the certificate expiration date.`,
+									References: []string{"https://aws.amazon.com/blogs/database/amazon-rds-customers-update-your-ssl-tls-certificates-by-february-5-2020/"},
+									Resources: []report.ResourcesGroup{{
```

Comment:
This is totally a matter of taste, but I'd separate the `{` on different lines, like:
```
Resources: []report.ResourcesGroup{
    {
    ...
    },
}
```

Reply:
Sure